frontend/src/components/UserDashboard/ProgressNotes/Note.tsx

```
@@ -67,13 +80,10 @@ const Note: React.FC<NoteProps> = ({
           </div>
         </div>
-        <div className="flex gap-2 flex-wrap">
-          {progressTrackers.map(({ label, value }, i) => (
-            <ProgressTracker key={i} label={label} value={value} />
-          ))}
-        </div>
 
-        {!!content.length && <div className="pt-2">{content}</div>}
+        <Separator />
+
+        {!!content.length && <p className="p-3" dangerouslySetInnerHTML={{ __html: content }}></p>}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)